css/index.css

```diff
@@ -1 +1,383 @@
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    font-size: 77px;
+    margin-top: 10%;
+    text-align: center;
+    padding: 0px 10%;
+}
+
+.cta .cta-text button {
+	display: flex;
+	align-self: center;
+    background: white;
+    margin-top: 4%;
+    padding: 2% 8%;
+    font-size: 17px;
+    cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+    background: black;
+    color: white;
+}
+
+.cta img {
+    margin-top: 7%;
+}
+
+.main-content {
+	display: flex;
+	flex-direction: column;
+    border-top: 1px solid black;
+    margin-top: 9%;
+    padding-top: 5%;
+}
+
+.main-content .top-content {
+  padding: 0 1%;
+  display: flex;
+}
+
+.main-content .top-content .text-content {
+  width: 50%;
+  margin-right: 4%;
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  margin-top: 5%;
+}
+
+.main-content .bottom-content {
+  margin-top: 4%;
+  padding: 0 1%;
+  display: flex;
+}
+
+.main-content .bottom-content .text-content {
+  width: 35%;
+  margin-right: 2%;
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  margin-top: 5%;
+  border-top: 1px solid black;
+  padding: 3% 1% 0;
+}
+
+.contact p {
+  margin: 2% 0;
+}
+
+footer {
+  margin: 3% 0;
+  text-align: center;
+  font-size: 14px;
+}
+
+.top-part {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	height: 30%;
+	border-bottom: 1px solid black;
+	margin-top: 2%;
+	margin-left: 3%;
+}
+
+.top-part h1 {
+	display: flex;
+	margin-top: 2%;
+}
+
+.mainbody {
+	display: flex;
+	flex-wrap: wrap;
+	align-content: center;
+	justify-content: center;
+	margin-left: 30px;
+}
+
+.mainbody div{
+	background: lightgray;
+	display: flex;
+	flex-direction: column;
+	align-content: space-between;
+	flex-wrap: wrap;
+	border: 1px black solid;
+}
+
+.mainbody button{
+	background: white;
+	border-radius: 5px;
+	border: 1px solid black;
+	width: 100px;
+	height: 30px;
+}
+
+.bottom {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	border-top: 2px solid black;
+	flex-wrap: wrap;
+
+}
+
+.bottom .partone {
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+}
+
+
+
+
+/* TABLET VIEW */
+@media (max-width: 800px) {
+    .container {
+        width: 100%;
+    }
+
+    .cta .cta-text {
+        align-self: auto;
+        margin-left: 18%;
+    }
+    .cta .cta-text button {
+        align-self: auto;
+        margin-left: 15%;
+    }
+    .cta img {
+        display: none;
+    }
+    .container header {
+        display: flex;
+        flex-direction: column-reverse;
+    }
+    .container header a {
+        margin: 3%;
+    }
+
+    .container .cta {
+        display: flex;
+        justify-content: center;
+        width: 100%;
+    }
+}
+
+@media (max-width: 500px) {
+    .container {
+        width: 100%;
+    }
+    .container header {
+
+        display: flex;
```

Comment:
Remember the cascading principle of CSS, there is no need to repeat yourself with `display: flex` here, as it was set to that earlier (higher) in the code.